### PR TITLE
Feature/bench 1964

### PR DESCRIPTION
**Pull request by aysn-capco**
## Ticket

*Add a link to the JIRA ticket here.*

## Description

*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this PR is changing it, and what motivated the change.*

## WIP

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).

*PRs that should not be reviewed should be marked as a WIP.

- [ ] This PR is currently a work-in-progress.
- [ ] This PR is ready for review.

## Checklist

- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors.
- [ ] All existing and new tests are passing.
- [ ] My PR meets the acceptance criteria as outlined by my ticket.
- [ ] I updated/added relevant documentation and comments (docs start with `///` and comments with `//`).
- [ ] Static analysis (e.g. Snyk) does not report any problems on my PR.

## Screenshots

*Screenshots can be added here.*

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint --fix --ext .js,.jsx ./src/."
   },
   "eslintConfig": {
```
Comment (daria-capco) [inline package.json:29]: you can move extends body to .eslintrc.json file and remove "eslintConfig" from package.json

**Review by daria-capco — CHANGES_REQUESTED** [review]
Good job!

File: package.json (hunk 1 above)
Comment (aysn-capco) [reply package.json]: alright , removed

Comment (daria-capco) [reply package.json]: I can see that you removed it instead of moving to .eslintrc didn't it break anything? did you try to run lint after it?

Comment (aysn-capco) [reply package.json]: yep

**Review by daria-capco — APPROVED** [review]

**Review by loka-capco — APPROVED** [review]

**Review by tbrj-capco — APPROVED** [review]
Looks good to me! Approved

**Pull request merged**